Makefile

```diff
@@ -1,3 +1,4 @@
+SHELL := /bin/bash
```

Comment:
interesting what os are you on?

I should test this on linux, i found oddly on macOS my default /bin/sh was bash (ubuntu does dash)

```
sh-3.2$ /bin/sh --help
GNU bash, version 3.2.57(1)-release-(arm64-apple-darwin22)
Usage:  /bin/sh [GNU long option] [option] ...
        /bin/sh [GNU long option] [option] script-file ...
GNU long options:
        --debug
        --debugger
        --dump-po-strings
        --dump-strings
        --help
        --init-file
        --login
        --noediting
        --noprofile
        --norc
        --posix
        --protected
        --rcfile
        --restricted
        --verbose
        --version
        --wordexp
Shell options:
        -irsD or -c command or -O shopt_option          (invocation only)
        -abefhkmnptuvxBCHP or -o option
Type `/bin/sh -c "help set"' for more information about shell options.
Type `/bin/sh -c help' for more information about shell builtin commands.
Use the `bashbug' command to report bugs.
```

Reply:
I'm on Ubuntu 22.04, where apparently `sh` points to `dash` rather than `bash`:

```console
❯ echo $SHELL
/usr/bin/fish

❯ sh --help
sh: 0: Illegal option --

~/code on ☁️  ajsbx (us-east-2) 
❯ ll (which sh)
lrwxrwxrwx 1 root root 4 Mar 23  2022 /usr/bin/sh -> dash
```